src/cli/commands/init.js

```diff
@@ -135,3 +134,16 @@ export async function run(
 
   await config.saveRootManifests(manifests);
 }
+
+
+async function getAuthorData(
+  reporter: Reporter,
+  credential: string,
+): Promise<string> {
+  try {
+    // try to get author default based on git config
+    return await child.spawn('git', ['config', credential]);
+  } catch (e) {
+    return await reporter.question(credential.replace('user.', 'author\'s '));
```

Comment:
I think they prefer to double quote when there is a single quote instead of escaping the single quote.